failing test: test/util.test.js

```
 FAIL  test/util.test.js
  ● Test suite failed to run

    Cannot find module '../src/util' from 'test/util.test.js'

      1 | /* global describe, it */
      2 | const assert = require('assert')
    > 3 | const buildOptions = require('../src/util')
        |                      ^
      4 |
      5 | describe('buildOptions', () => {
      6 |   it('returns an empty string if the array passed to it is empty', () => {

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.require (test/util.test.js:3:22)
```

--- a/test/util.test.js
+++ b/test/util.test.js
@@ -1,6 +1,6 @@
 /* global describe, it */
 const assert = require('assert')
-const buildOptions = require('../src/util')
+const buildOptions = require('../src/internal/util')
 
 describe('buildOptions', () => {
   it('returns an empty string if the array passed to it is empty', () => {

